lib/modules/manager/buildkite/extract.ts

```diff
@@ -13,7 +13,7 @@ export function extractPackageFile(content: string): PackageFile | null {
     for (const line of lines) {
       // Search each line for plugin names
       const depLineMatch = regEx(
-        /^[\s-]*(?<depName>[^#\s]+)#(?<currentValue>[^:]+)/
+        /^[\s-?]*(?<depName>[^#\s]+)#(?<currentValue>[^:]+)/
```

Comment:
seems wrong to me. regex allows ` - ? - ? ...`.
should be more strict.

Reply:
I can make it more strict!